.github/workflows/image-build.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright contributors to the Hyperledger Fabric Operations Console project
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+# 	  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Build and Push Deployer image
+on:
+  push:
+    tags:
+      - '*'
+env:
+  GOPATH: /opt/go
+  PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GO_VER: 1.23.5
+
+jobs:
+  build-deployer-image:
+    name: Build and Push Deployer image
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/fabric-deployer
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/setup-go@v5
+        name: Install Go
+        with:
+           go-version: ${{ env.GO_VER }}
+      - name: Checkout fabric-deployer repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Check Go version
+        run: go version
+      - name: List repository files
+        run: ls -la
+      - name: Build
+        run: make image
+      - name: Push
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
+          make image-push image-push-latest
```

Comment:
It looks like the image name is still ghcr.io/ibm-blockchain/fabric-deployer at:
https://github.com/hyperledger-labs/fabric-operations-console/blob/main/packages/fabric-deployer/Makefile#L19

Reply:
Updated